graphql-dgs-reactive/src/main/kotlin/com/netflix/graphql/dgs/reactive/internal/HandlerMethodArgumentResolverAdapter.kt

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2022 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.graphql.dgs.reactive.internal
+
+import com.netflix.graphql.dgs.context.DgsContext
+import com.netflix.graphql.dgs.internal.ArgumentResolver
+import graphql.schema.DataFetchingEnvironment
+import org.springframework.core.MethodParameter
+import org.springframework.web.reactive.BindingContext
+import org.springframework.web.reactive.result.method.SyncHandlerMethodArgumentResolver
+
+class HandlerMethodArgumentResolverAdapter(
```

Comment:
In Spring-Webflux, it's literally called a `HandlerMethodArgumentResolver`, I think calling it anything else would be confusing. When i add the docs here I'll link to the actual classes, so it'll be clearer.

Reply:
Actually, since I only support `SyncHandlerMethodArgumentResolver` here, if I were to rename it I'd probably rename it to `SyncHandlerMethodArgumentResolverAdapter`.

Reply:
Ok, that makes sense.